tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -404,3 +1174,279 @@ def test_quentin_profiler_user_workflow_multi_batch_quantiles_value_ranges_rule(
+    )
+    assert len(validator.batches) == 36
+
+    # Use all batches, loaded by Validator, for estimating Expectation argument values.
+    result = validator.expect_column_quantile_values_to_be_between(
+        column="fare_amount",
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+    )
+    assert result.success
+
+    key: str
+    value: Any
+    expectation_config_kwargs: dict = {
+        key: value
+        for key, value in result.expectation_config["kwargs"].items()
+        if key != "quantile_ranges"
+    }
+    assert expectation_config_kwargs == {
+        "column": "fare_amount",
+        "allow_relative_error": "linear",
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "batch_id": "84000630d1b69a0fe870c94fb26a32bc",
+    }
+
+    # Measure of "closeness" between "actual" and "desired" is computed as: atol + rtol * abs(desired)
+    # (see "https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html" for details).
+    rtol: float = 1.0e-7
+    atol: float = 5.0e-2
+
+    value_ranges_expected: List[List[float]]
+    value_ranges_computed: List[List[float]]
+
+    value_ranges_expected = [
+        [5.8, 6.5],
+        [8.5, 9.7],
+        [13.3, 15.8],
+    ]
+    value_ranges_computed = result.expectation_config["kwargs"]["quantile_ranges"][
+        "value_ranges"
+    ]
+
+    assert len(value_ranges_computed) == len(value_ranges_expected)
+
+    paired_quantiles: zip
+    value_ranges: Tuple[List[float]]
+    idx: int
+
+    paired_quantiles = zip(
+        value_ranges_computed,
+        value_ranges_expected,
+    )
+    for value_ranges in list(paired_quantiles):
+        for idx in range(2):
+            np.testing.assert_allclose(
+                actual=value_ranges[0][idx],
+                desired=value_ranges[1][idx],
+                rtol=rtol,
+                atol=atol,
+                err_msg=f"Actual value of {value_ranges[0][idx]} differs from expected value of {value_ranges[1][idx]} by more than {atol + rtol * abs(value_ranges[1][idx])} tolerance.",
+            )
+
+    parameter_builder_batch_request: dict
+    custom_profiler_config: dict
+
+    # Use one batch (at index "1"), loaded by Validator as active_batch, for DomainBuilder purposes.
+    domain_batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {
+            "index": -1,
+        },
+    }
+
+    # Use all batches, except ("latest") active_batch, loaded by Validator, for estimating Expectation argument values.
+    parameter_builder_batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {
+            "index": ":-1",
+        },
+    }
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner Again -- this is a custom override, provided by users (who may find plain dictionary more convenient -- at least we have held this pattern for user-facing `batch_request` API.

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!